mdoc/src/main/scala/mdoc/internal/markdown/Renderer.scala

```diff
@@ -24,8 +24,11 @@ object Renderer {
       filename: String,
       printer: Variable => String
   ): String = {
+    val sbt1WithLiteralTypes = dialects.Sbt1.copy(allowLiteralTypes = true)
```

Comment:
Let's reuse the same dialect here and in `Processor`

Reply:
I've created a `MdocDialect` object in `Processor` that gets reused in in `Renderer`, I'm not sure if that's what you had in mind though.